packable/packable/tests/common/mod.rs

```diff
@@ -52,15 +52,15 @@ where
 
     let mut vec = Vec::new();
     packable.pack(&mut vec).unwrap();
-    let unpacked = P::unpack_verified(vec.as_slice(), &()).unwrap();
+    let unpacked = P::unpack_bytes_verified(vec.as_slice(), &()).unwrap();
     assert_eq!(packable, &unpacked);
 
     // Tests for `Read` and `Write`
 
     let mut packer = IoPacker::new(Vec::new());
     packable.pack(&mut packer).unwrap();
     let mut unpacker = IoUnpacker::new(packer.as_slice());
-    let unpacked = P::unpack::<_, true>(&mut unpacker, &()).unwrap();
+    let unpacked = P::unpack(&mut unpacker, None).unwrap();
```

Comment:
Before this had `true`, wouldn't the equivalent now be `Some(())`?

Reply:
actually it would be `unpack_inner`. I missed it, thanks

Reply:
actually,  I was wrong in this case as the visitor type is always `()`, but anyway I fixed it